pkg/apis/v1beta1/nodepool_budgets_test.go

```diff
@@ -70,43 +70,45 @@ var _ = Describe("Budgets", func() {
 			},
 		}
 	})
-	Context("NodePool/AllowedDisruptions", func() {
+	Context("NodePool/MustGetAllowedDisruptions", func() {
```

Comment:
Do these `Context` blocks need to have `NodePool` in front of them? This is all NodePool testing so I think it's fine to just say `MustGetAllowedDisruptions`